.github/ISSUE_TEMPLATE/feature-request.yaml

```diff
@@ -0,0 +1,49 @@
+name: Feature request
+description: Request a new feature
+type: Feature
+body:
+  - type: markdown
+    id: introduction
+    attributes:
+      value: |
+        ### Thank you for taking the time to suggest a feature!
+
+  - type: checkboxes
+    id: before-posting
+    attributes:
+      label: "This feature request respects the following points:"
+      description: Please help us by ensuring that you meet all the following conditions.
+      options:
+        - label: This feature has **not** already been requested on [GitHub](https://github.com/jellyfin/Swiftfin/issues?q=is%3Aissue%20state%3Aopen%20type%3AFeature).
+          required: true
+        - label: I agree to follow Jellyfin's [Code of Conduct](https://jellyfin.org/docs/general/community-standards.html#code-of-conduct).
+          required: true
+
+  - type: checkboxes
```

Comment:
I'm not worried about this checkbox, as I only consider a PR to be the ultimate signal of an intent to contribute.